### PR TITLE
Reviews branch

### PR DESCRIPTION
**Pull request by emilyasaro**
Created Review Model and seeded 1 review. Have not yet made a test for reviews.
Created and passed a test for /api/orders

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const OrderDetail = require('./models/shopping');
 const User = require('./models/users/user');
```
Comment (JonathanCrider) [inline server/db/index.js:2]: should this be Users? doesn't Sequelize pluralize the model name?

**Review by JonathanCrider — CHANGES_REQUESTED** [review]
update your model name imports to match what Sequelize does to your model, so User = Users, etc

**Review by emilyasaro — COMMENTED** [review]
Resolved

**Review by JonathanCrider — APPROVED** [review]
sorry i'm dumb

**Pull request merged**